packages/sources/coingecko/src/util.ts

```diff
@@ -1,7 +1,6 @@
 import { endpointSelector, makeExecute } from './adapter'
 import { makeMiddleware, withMiddleware } from '@chainlink/ea-bootstrap'
-import { CoinsResponse } from './endpoint/coins'
```

Comment:
Won't `CoinsResponse` vary per DP?

Reply:
Whatever the DP provider spits out, we coerce that to fit the CoinsResponse type.  I verified this behavior in CoinPaprika too